habitat-baselines/habitat_baselines/rl/multi_agent/measures.py

```diff
@@ -42,6 +42,30 @@ def update_metric(self, *args, task, **kwargs):
         self._metric = found_contact
 
 
+@registry.register_measure
+class NumCollisions(Measure):
```

Comment:
Hey @ASzot, shouldn't this measure be named NumAgentsCollide?

Reply:
Will fix and make a new PR, thanks for catching @rpartsey !